events/importer/lippupiste.py

```diff
@@ -0,0 +1,371 @@
+
+import codecs
+import csv
+import pytz
+import re
+import bleach
+import requests
+from datetime import datetime
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+from django.utils.html import strip_tags
+from django_orghierarchy.models import Organization
+from events.models import DataSource, Event, Keyword, Place
+from .base import Importer, recur_dict, register_importer
+from .sync import ModelSyncher
+from .util import clean_text
+
+
+YSO_KEYWORD_MAPS = {
+    'baletti': ('yso:p1278', 'yso:p10218'),
+    'dance': ('yso:p1278',),
+    'draama': ('yso:p2625',),
+    'elokuva': ('yso:p1235',),
+    'elokuvafestivaali': ('yso:p1304', 'yso:p1235'),
+    'elämänhallinta': ('yso:p4357',),
+    'farssi, satiiri': ('yso:p2625',),
+    'hard rock': ('yso:p1808', 'yso:p1882', 'yso:p29778'),
+    'hard rock -festivaali': ('yso:p1304', 'yso:p1808', 'yso:p1882', 'yso:p29778'),
+    'iskelmä- ja tanssimusiikki': ('yso:p1808', 'yso:p1857', 'yso:p181'),
+    'jalkapallo': ('yso:p965', 'yso:p6409'),
+    'jazz & blues': ('yso:p1808', 'yso:p4484', 'yso:p4482'),
+    'jazz & blues -festivaali': ('yso:p1304', 'yso:p1808', 'yso:p4484', 'yso:p4482'),
+    'jääkiekko': ('yso:p965', 'yso:p12697'),
+    'kabaree': ('yso:p1808', 'yso:p181', 'yso:p7158'),
+    'kansanmusiikki': ('yso:p1808', 'yso:p2841'),
+    'kesäteatteri': ('yso:p2625', 'yso:p17654'),
+    'klassinen musiikki': ('yso:p1808', 'yso:p18434'),
+    'klassisen musiikin festivaali': ('yso:p1304', 'yso:p1808', 'yso:p18434'),
+    'klubit': ('yso:p1808', 'yso:p20421'),
+    'komedia': ('yso:p2625', 'yso:p13876'),
+    'koripallo': ('yso:p965', 'yso:p8781'),
+    'lastennäytelmä': ('yso:p2625', 'yso:p16164'),
+    'musikaali, musiikkiteatteri': ('yso:p1808', 'yso:p11693', 'yso:p6422'),
```

Comment:
I would add `yso:p2625` here so that also musicals will be listed under theatre.